### PR TITLE
Initial doc setup

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

File: doc/model.rst [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+.. _heat-flux-constitutive:
+
+Heat flux constitutive relation
+-------------------------------
+
+The heat flux vector is defined as
+
+\f[
+q_i = - \kappa \frac{\partial T}{\partial x_i}
+\f]
```
Comment (lukeolson) [inline doc/model.rst:56]: What is `\f]`?  Is there some math/rst documentation to point to in a readme?

File: doc/model.rst [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+the non-dimensional viscous stress tensor and heat flux vector can be written as
+
+.. math::
+    \tau_{ij} &= \frac{\mu}{\RE} \left(\frac{\partial u_i}{\partial x_j} +
+      \frac{\partial u_j}{\partial x_i}\right) +
+      \frac{\lambda}{\RE} \frac{\partial u_k}{\partial x_k}\delta_{ij} \\
+    q_i &= - \frac{\mu}{\RE \Pr} \frac{\partial T}{\partial x_i}
+
+where :math:`\RE` is defined as the code Reynolds number,
```
Comment (lukeolson) [inline doc/model.rst:188]: `\RE` fails in `latexpdf`.  How can we add a `newcommand` in rst?

File: doc/model.rst (hunk 1 above)
Comment (inducer) [reply doc/model.rst]: Doxygen leftovers, fixed in e0cf898.

File: doc/model.rst (hunk 2 above)
Comment (inducer) [reply doc/model.rst:199]: 6b92ef4

**Comment by inducer** [comment]
@lukeolson I added some stub development docs here.

File: doc/conf.py [hunk 3]
```diff
@@ -43,6 +43,8 @@
         "sphinx.ext.autodoc",
         "sphinx.ext.doctest",
         "sphinx.ext.napoleon",
+        "sphinx.ext.graphviz",
```
[the remark this answers is not in the document]
Comment (inducer) [reply doc/conf.py:46]: It's a trade-off. It means we get to use graphviz for quick diagrams in the docs, but it increases the barrier to building the docs very slightly. If we can just document (in the documentation for how to build the docs, 99386a2) to install graphviz at the same time as sphinx, I think it won't hurt too much.

**Comment by inducer** [comment]
@lukeolson Thanks for the review!

**Pull request merged**